reverseCase/reverseCase.js

```diff
@@ -2,4 +2,20 @@
  * Write a function that reverses the case of each letter in the strings that it receives.
  * Example: 'Hello World' -> 'hELLO wORLD'
  * Assume that each string will contain only spaces and letters.
- */
\ No newline at end of file
+ */
+const reverseCase = (string) => {
+  let newString = '';
+  for (let i =0; i < string.length; i++) {
+    if(string[i] === string[i].toUpperCase()) {
+      newString += string[i].toLowerCase();
+    } else if (string[i] === string[i].toLowerCase()) {
+      newString += string[i].toUpperCase();
+    } else {
```

Comment:
I'm trying to figure out why you need a third condition and not just the first two. This would imply that you're input string is something more than lower and uppercase. Where this is a binary check, (upper vs lower case) you only really need the two. BUT if your given string contained numbers or other types of data, this would concatenate that data to the string.